demo_nodes_cpp/src/parameters/parameter_blackboard.cpp

```diff
@@ -29,12 +29,11 @@ class ParameterBlackboard : public rclcpp::Node
 public:
   DEMO_NODES_CPP_PUBLIC
   explicit ParameterBlackboard(
-    const rclcpp::NodeOptions & options = (
-      rclcpp::NodeOptions()
-      .allow_undeclared_parameters(true)
-      .automatically_declare_parameters_from_overrides(true)
-  ))
-  : Node("parameter_blackboard", options)
+  rclcpp::NodeOptions options
+  )
+  : Node("parameter_blackboard",
+    options.allow_undeclared_parameters(true).
+    automatically_declare_parameters_from_overrides(true))
```

Comment:
The correct style is:
```cpp
  : Node(
      "parameter_blackboard",
      options.allow_undeclared_parameters(true).
      automatically_declare_parameters_from_overrides(true))
```

Reply:
got it, fixed in https://github.com/ros2/demos/pull/389/commits/2dc4fb39353d225d0aed9d06c5e4cea1009d6f83